pkg/helmexec/exec.go

```diff
@@ -546,3 +542,19 @@ func (helm *execer) IsVersionAtLeast(versionStr string) bool {
 	ver := semver.MustParse(versionStr)
 	return helm.version.Equal(ver) || helm.version.GreaterThan(ver)
 }
+
+func resolveOciChart(ociChart string) (string, string) {
+	var urlTagIndex int
+	//Get the last : index
+	//e.g., registry:443/helm-charts
+	//e.g., registry/helm-charts:latest
+	//e.g., registry:443/helm-charts:latest
```

Comment:
// Get the last : index
	// 1. registry:443/helm-charts
	// 2. registry/helm-charts:latest
	// 3. registry:443/helm-charts:latest

Reply:
add test for all case. Thanks very much.

Reply:
Submitted, Thanks.

Reply:
update comments. Thanks very much.